http-client/mhs/asynchronous-express-message-pattern/Return-information-from-ebxml-fault-returned-from-spine-in-original-post-request-when-sync-async-requested.http

```diff
@@ -0,0 +1,16 @@
+
+
+@PATIENT_NHS_NUMBER = 9689177923
+@DISSENTOVERRIDE = dissentOverride
+# @name Return_information_from_ebxml_fault_returned_from_spine_in_original_post_request_when_sync_async_requested
+POST http://localhost HTTP/1.1
+interaction-id: QUPC_IN160101UK05
+message-id: 7AA57E38-8B20-4AE0-9E73-B9B0C0C42BDA
+sync-async: true
+from-asid: {{ASID}}
+Content-Type: application/json
+ods-code: YES
+
+{
+    "payload": "<QUPC_IN160101UK05 xmlns=\"urn:hl7-org:v3\">\r\n<id root=\"{{GUID-idRoot}}\" />\r\n<creationTime value=\"{{SHORT-TIMESTAMP}}\"/>\r\n<versionCode code=\"3NPfIT7.2.00\" />\r\n<interactionId root=\"2.16.840.1.113883.2.1.3.2.4.12\" extension=\"QUPC_IN160101UK05\" />\r\n<processingCode code=\"P\" />\r\n<processingModeCode code=\"T\" />\r\n<acceptAckCode code=\"NE\" />\r\n<communicationFunctionRcv typeCode=\"RCV\">\r\n<device classCode=\"DEV\" determinerCode=\"INSTANCE\">\r\n<id extension=\"{{PARTY-KEY}}\" root=\"1.2.826.0.1285.0.2.0.107\" />\r\n</device>\r\n</communicationFunctionRcv>\r\n<communicationFunctionSnd typeCode=\"SND\">\r\n<device classCode=\"DEV\" determinerCode=\"INSTANCE\">\r\n<id extension=\"{{ASID}}\" root=\"1.2.826.0.1285.0.2.0.107\" />\r\n</device>\r\n</communicationFunctionSnd>\r\n<ControlActEvent classCode=\"CACT\" moodCode=\"EVN\">\r\n<author1 typeCode=\"AUT\">\r\n<AgentSystemSDS classCode=\"AGNT\">\r\n<agentSystemSDS classCode=\"DEV\" determinerCode=\"INSTANCE\">\r\n<id extension=\"{{ASID}}\" root=\"1.2.826.0.1285.0.2.0.107\" />\r\n</agentSystemSDS>\r\n</AgentSystemSDS>\r\n</author1>\r\n<query>\r\n<dissentOverride>\r\n<semanticsText>DissentOverride</semanticsText>\r\n<value code=\"{{DISSENTOVERRIDE}}\" codeSystem=\"2.16.840.1.113883.2.1.3.2.4.17.60\" displayName=\"Demonstration\">\r\n<originalText>Demonstration</originalText>\r\n</value>\r\n</dissentOverride>\r\n<filterParameters>\r\n<date>\r\n<semanticsText>Date</semanticsText>\r\n<value>\r\n<low value=\"20100908161126\"/>\r\n<high value=\"{{SHORT-TIMESTAMP}}\"/>\r\n</value>\r\n</date>\r\n</filterParameters>\r\n<nHSNumber>\r\n<semanticsText>NHSNumber</semanticsText>\r\n<value root=\"2.16.840.1.113883.2.1.4.1\" extension=\"{{PATIENT_NHS_NUMBER}}\" />\r\n</nHSNumber>\r\n</query>\r\n</ControlActEvent>\r\n</QUPC_IN160101UK05>\r\n"
```

Comment:
Is it possible to reformat this so the XML inside is "pretty printed"? Or put it into a separate file. I think VSCode HTTP Client has ability to load variables from file content

Reply:
Not possible to have on multiple lines, error json body. Cannot use environment variables though.